docs/src/intro-js.md

```diff
@@ -102,6 +102,13 @@ npx playwright show-report
 
 <img width="1392" alt="HTML Reporter" src="https://user-images.githubusercontent.com/13063165/212743312-edf1e8ed-3fc2-48aa-9c93-24ae3e36504d.png" />
 
+## System requirements
+
+- Node.js 16 or Node.js 18.
```

Comment:
```suggestion
- Node.js 16+
```
maybe that? What about Node.js 20?

Reply:
Last time I checked we didn't support Node 20 😄  Do we now?

Reply:
We do!